src/vcpkg/postbuildlint.cpp

```diff
@@ -890,10 +903,15 @@ namespace vcpkg::PostBuildLint
         const auto debug_bin_dir = package_dir / "debug/bin";
         const auto release_bin_dir = package_dir / "bin";
 
+#ifdef __APPLE__
+        NotExtensionsCaseInsensitive libFilter{{".so", ".a", ".dylib"}};
+#else
+        NotExtensionCaseInsensitive libFilter{".lib"};
+#endif
```

Comment:
```suggestion
#ifdef __APPLE__
        NotExtensionsCaseInsensitive lib_filter{{".so", ".a", ".dylib"}};
#else
        NotExtensionCaseInsensitive lib_filter{".lib"};
#endif
```

Reply:
This should also probably check if the target is osx or windows, not if the host is osx.